lib/super_digit.rb

```diff
@@ -1,15 +1,60 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log k*n), but it really feels like O(log n):
+#                              # of times you sum the digits is a operation dependent on the number of
```

Comment:
👍  However the time complexity is O(n^2) where n is the number of digits, you can see why [here](https://learnersbucket.com/examples/algorithms/find-digital-root-of-a-number/)

Reply:
Thank you so much for the links!!!